nodejs/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM socrata/base
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install nodejs npm
```

Comment:
the apt-get update and install need to be part of the same RUN command to avoid potential dependency problems.